pages/guides/how-to-schedule-price-updates-with-gelato.mdx

```diff
@@ -0,0 +1,125 @@
+# How to Schedule Price Updates with Gelato
+
+This guide explains how to create an automated task that updates Pyth price feeds on a schedule
+
+### Why Gelato? Introduction to Gelato
+
+Gelato is web3’s decentralized backend empowering builders to create augmented smart contracts that are automated,
+gasless & off-chain aware on all major EVM-compatible blockchains. Gelato offers Web3 Functions (W3F) that connect your
+smart contracts to off-chain data & computation by running decentralized cloud functions
+
+### Gelato W3F Pyth Task
+
+The Gelato W3F Pyth Task is a Gelato task that updates Pyth price feeds on a schedule.
+The task is deployed to IPFS and can be used by anyone to schedule price updates for any Pyth price feed.
+The task is configured via a yaml file that is hosted on GitHub gist which contains
+various parameters such as the price feeds to update, the update frequency,
+price deviation threshold and can be dynamically updated at any time.
+
+## Steps
+
+#### 1. Join Gelato Web3 Functions whitelist
+
+Web3 Functions are currently in private beta.
+Get whitelisted in the Web3 Function private Beta. If you are not, reach out to
+the Gelato team [here](https://form.typeform.com/to/RrEiARiI).
+
+#### 2. Configure your task
+
+The Gelato task reads a yaml configuration file from a GitHub gist.
+Create a GitHub gist and add a file called `config.yaml`.
+The `config.yaml` file should follow the example `config.yaml` files from this
+
+[//]: # "TODO: redirect this to pyth fork of repo)"
+
+[directory](https://github.com/gelatodigital/w3f-pyth-poc-v2/tree/master/web3-functions/pyth-oracle-w3f-priceIds)
+[example gist](https://gist.github.com/swimricky/18b2a5ad9c1a605f1cf5c19ac1d2f1d9)
+
+Once the task is running, you can update this configuration to add more price feeds or change the update parameters.
```

Comment:
maybe this should start with something like "Don't worry if you don't know the exact configuration parameters that you want." 

(I think that's the underlying reason you have this text here, but it's not clear)